data/map.txt

```diff
@@ -29124,6 +29246,35 @@ planet "New Switzerland"
 		threshold 2000
 		fleet "Small Republic" 11
 
+planet "New Tortuga"
+	attributes pirate "core pirate" core religious factory
+	landscape land/fog6
+	description `A planet of violent storms and frequent earthquakes, New Tortuga would turn away anyone unless they have a reason to hide. The few settlements on the planet are subterranean cave cities far from any fault lines. Thick blast doors block all entrances, and vast pumping systems remove water after the frequent supertyphoon-strength tropical storms.`
+	description `Most of the inhabitants of this planet are Syndicate dissidents who support themselves through manufacturing. The Syndicate often destroys pirate factories at their borders, but their fleets cannot penetrate as far as New Tortuga. Here, shipyards, factories, and outfitters provide an uninterrupted supply.`
```

Comment:
I liked much of this one as is. I didn't know if you wanted a complete overhaul with different facts or tidbits, but I only changed small things about this one. 

Alternative Landscape (2) Description 1: "Most of the inhabitants of this planet are Syndicate dissidents who mainly support themselves through manufacturing. The Syndicate often destroys pirate factories at their borders, but their fleets do not find the monetary cost of penetrating as far as New Tortuga worth the benefit of a few dead pirates. Here, shipyards, factories, and outfitters provide an uninterrupted supply of goods and services for the system."

Alternative Landscape (2) Description 2: "Most of the inhabitants of this planet are Syndicate dissidents who mainly support themselves through manufacturing ships and outfits. The Syndicate often destroys pirate factories at their borders, but their fleets do not find the monetary cost of venturing as far as New Tortuga worth the benefit of a few dead pirates. Here, shipyards, factories, and outfitters provide an uninterrupted supply of goods and services for the system."